molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
```

Comment:
Either delete this test (and file) or replace it with a basic smoke test. Also ensure that the corresponding `test_default.pyc` is removed from this PR.

Reply:
Wil be removed in the next push